libs/deploy.js

```diff
@@ -0,0 +1,16 @@
+const utils = require('./utils');
+
+const DEPLOYMENT_DELAY = 1500;
```

Comment:
what's this for?

Reply:
There is a delay from the time the package is `deployed` to when its actually available to invoke (for new versions). From my testing I noticed tops it was around 1.5 seconds, but usually is less than 1 second.